backend/pkg/sqlmanager/postgres/postgres-manager.go

```diff
@@ -875,7 +875,7 @@ func (p *PostgresManager) GetTableRowCount(
 func BuildPgTruncateStatement(
 	tables []string,
 ) string {
-	return fmt.Sprintf("TRUNCATE TABLE %s;", strings.Join(tables, ", "))
+	return fmt.Sprintf("TRUNCATE TABLE %s RESTART IDENTITY;", strings.Join(tables, ", "))
```

Comment:
small side note, but interesting that this one builds it manually, but the cascade statement builds it using goqu 😄

Reply:
will update this as a fast follow